pkgs/modules/bundles/python/default.nix

```diff
@@ -0,0 +1,13 @@
+{ pkgs, config, ... }:
+pkgs.lib.mkBundleModule {
+  id = "python";
+  name = "Python Tools Bundle";
+  description = "Development tools for the Python 3 runtime";
+  submodules = [
+    "interpreters.python"
+    "languageServers.pyright-extended"
+    "packagers.python"
+    "debuggers.debugpy"
+  ];
```

Comment:
hm, so overlang has `.` in some of its attrs. for example:
```
pkgs.nodejsVersions."20.2.0"
```

i'm wondering if there's a scenario where these patterns will conflict? not something that needs to be addressed now, just a thought

Reply:
Why did you bring it up wrt to submodules? The python bundle tries to not be concerned with version and let that be set at the python interpreter level.